spec/system/support/precompile_assets.rb

```diff
@@ -14,7 +14,7 @@
     begin
       # Silence NPM build output
       $stdout.reopen(File.new("/dev/null", "w"))
-      system("npm build")
+      system("npm run build")
```

Comment:
The command was broken, I know you are changing to CSS bundling and JS bundling but the `tailwind-build.css` was never built when I tested this.